README.md

```diff
@@ -126,9 +134,11 @@ e.g. `docker run -v /config:/config  ghcr.io/caas-team/sparrow --config /config/
 
 ## Configuration
 
-The configuration is divided into two parts. The startup configuration and the runtime configuration. The startup
-configuration is a technical configuration to configure the `sparrow` instance itself. The runtime configuration will be
-loaded by the `loader` from a remote endpoint. This configuration consists of the checks' configuration.
+The configuration is divided into two parts. The startup configuration and the checks configuration. The startup
+configuration is a technical configuration to configure the `sparrow` instance itself.
+
+The `loader` fetches the checks configuration during runtime from a remote endpoint. For local use, you may directly load the configuration using a `file` loader.
```

Comment:
I would first explain what the config for the checks is and then mention how it is loaded. We could also only mention that in the specific section for the checkconfig.

Reply:
I've moved this [here](https://github.com/caas-team/sparrow/pull/76/files/5d88dff438baf80531224eef455c366e2dba59c8..e4aed56a4bfb9843c1b0976b873c42f99ffb1ecc#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R241)